components/engine_traits/src/util.rs

```diff
@@ -21,3 +29,192 @@ pub fn check_key_in_range(
         })
     }
 }
+
+lazy_static! {
+    static ref SEQUENCE_NUMBER_COUNTER_ALLOCATOR: AtomicU64 = AtomicU64::new(0);
```

Comment:
Why need lazy_static to make it static?

Reply:
Previously some variables were inited with CFs, forgot to remove lazy_static after changing the init way.